src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -470,6 +488,13 @@ export default function Swap({
         onConfirm={handleConfirmTokenWarning}
         onDismiss={handleDismissTokenWarning}
       />
+      <TransactionConfirmationModal
+        attemptingTxn={true}
+        isOpen={showTransactionConfirmationModal}
+        pendingText="Approving token for trading"
```

Comment:
Could be nice to pass the dynamic token symbol string here: ==> `Approving WETH for trading` or shorter `Approving WETH`